plugins/hydra_ax/README.md

```diff
@@ -1,7 +1,19 @@
-# Hydra example Launcher plugin
+# Hydra Ax Launcher plugin
 
-This plugin provides an example for how to write a custom Sweeper for Hydra.
+This plugin provides a mechanism for Hydra applications to use the [Adaptive Experimentation Platform aka Ax](https://ax.dev/). Ax can be used to optimize any kind of experiment - machine learning experiments, A/B tests and simulations. 
+
+The plugin can be installed by running the following command in the `plugins/hydra_ax` directory:
+
+```
+python setup.py install
```

Comment:
In general, do not use setup.py to install.
install with 
```bash
# current directory
pip install .
# in editable (develop) mode:
pip install -e .

# or package:
pip install hydra-ax
```
For this doc, please document the package method.

Reply:
To use the package method, do we not have to upload the package to pypi ?

Reply:
Yes, and I will.